### DISCUSSION TITLE
Only list git repos

### DISCUSSION THREAD

**Discussion by nbbaier** [eza-community/eza#904] [Q&A, Howto, Help] [answered]
Is it possible to filter directories that are not git repos?

**Comment by gierens** [comment, marked as answer]
Not directly with `eza` at the moment. You could do something like `eza -l --git-repos | grep '- -'`. Or open a feature request for this.

**Reply by nbbaier** [reply]
The following worked perfectly
```
eza -l --git-repos | grep -- "- -"
```
I can just assign that to an alias. Thanks!
